src/main/java/org/spongepowered/common/config/SpongeConfig.java

```diff
@@ -473,6 +477,14 @@ public boolean chunkLoadOverride() {
         public void setChunkLoadOverride(boolean chunkLoadOverride) {
             this.chunkLoadOverride = chunkLoadOverride;
         }
+
+        public Locale getLocale() {
+            return Locale.forLanguageTag(this.locale);
```

Comment:
Why should we restrict it to MC locales? This is all done on the server so there's really no need for that.